pkg/serverless/invocationlifecycle/init.go

```diff
@@ -85,6 +87,8 @@ func (lp *LifecycleProcessor) initFromSQSEvent(event events.SQSEvent) {
 
 }
 
-func (lp *LifecycleProcessor) initFromLambdaFunctionURLEvent(event events.LambdaFunctionURLRequest) {
+func (lp *LifecycleProcessor) initFromLambdaFunctionURLEvent(event events.LambdaFunctionURLRequest, region string, accountID string, functionName string) {
 	lp.addTag("function_trigger.event_source", "lambda-function-url")
+	lp.addTag("function_trigger.event_source_arn", fmt.Sprintf("arn:aws:lambda:%v:%v:url:%v", region, accountID, functionName))
```

Comment:
maybe we should be consistent with other type with a `trigger.ExtractxxxARN` function?

This would also unit testing this piece of logic while abstracting it from the invocationLifecycle package?

Reply:
I thought so too and I had that originally but it's just a sprintf statement that doesn't actually depend on the event, so it's not "extracting" anything. I think it makes more sense to keep it here :+1: